rsearch/doc.go

```diff
@@ -0,0 +1,3 @@
+// This package prvides means of searching kubernets objects by their selectors
```

Comment:
Spelling, and this becomes a single sentence in documentation, ie: it'll appear as "...selectors the goal is..."